components/IconButton.qml

```diff
@@ -47,28 +46,29 @@ Item {
         id: buttonImage
         source: ""
         x : (parent.width - width) / 2
-        y : (parent.height - height)  /2
+        y : (parent.height - height) / 2
         z: 100
     }
 
     MouseArea {
         id: buttonArea
-        anchors.fill: parent
+        anchors.fill: buttonImage
+        anchors.margins: -5
```

Comment:
```suggestion
```
This was the culprit :b

Reply:
That’s to make the clickable area a little big bigger than the icon. :)